docs/source/API/core/c_style_memory_management/malloc.md

```diff
@@ -18,6 +18,8 @@ If allocation succeeds, returns a pointer to the lowest (first) byte in the allo
 
 If allocation fails, an exception of type `Kokkos::Experimental::RawMemoryAllocationFailure` is thrown.
 
+WARNING: calling any function that manipulates the behavior of the memory (e.g. `memAdvise`) on memory managed by `Kokkos` results in undefined behavior. You are on your own with this.
+
```

Comment:
I would not include the second sentence, because the first communicates this fact.

Reply:
done